nesta/packages/misc_utils/s3_utils.py

```diff
@@ -0,0 +1,39 @@
+import pickle
+import boto3
+
+
+def store_on_s3(data, bucket, prefix):
```

Comment:
could you change the function names to `def pickle_to_s3` and `def s3_to_pickle` to avoid confusion with ascii files?